samples/common/mcumgr_bt_ota_dfu/Kconfig

```diff
@@ -134,3 +139,8 @@ config NCS_SAMPLE_MCUMGR_BT_OTA_DFU_VALIDATION
 	  Enable this option to validate the MCUmgr OTA DFU configuration at the
 	  build time. If the project configuration does not match predefined
 	  defaults, the build system issues a warning for each misaligned item.
+
+config NCS_SAMPLE_MCUMGR_BT_OTA_DFU_MCUBOOT_DIRECT_XIP_SUPPORT
+	bool
+	default y if MCUBOOT_BOOTLOADER_MODE_DIRECT_XIP
+	default y if MCUBOOT_BOOTLOADER_MODE_DIRECT_XIP_WITH_REVERT
```

Comment:
you can move implies here without any conditions to achieve the same effect

Reply:
done